pkg/gcp/build/build.go

```diff
@@ -404,14 +406,27 @@ func ListJobs(project string, lastJobs int64) error {
 		return fmt.Errorf("failed to listing the builds: %w", err)
 	}
 
-	table := tablewriter.NewWriter(os.Stdout)
-	table.SetHeader([]string{"Start Time", "Finish Time", "Status", "Console Logs"})
+	table := tablewriter.NewTable(os.Stdout,
```

Comment:
Can we add some function in release-utils that's going to return a table that's already configured with everything, so that we don't have to write this much code every time we want to use a table? Does that make sense at all?

Reply:
sg